packages/library/README.md

```
@@ -0,0 +1,144 @@
+[![License](https://img.shields.io/badge/License-APACHE2.0-blue.svg)](https://opensource.org/licenses/apache-2-0)
+[![License](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
+[![](https://tokei.rs/b1/github/j5ik2o/event-store-adapter-js)](https://github.com/XAMPPRocky/tokei)
+
+This library is designed to turn DynamoDB into an Event Store for CQRS/Event Sourcing.
+
+[日本語](./README.ja.md)
+
+# Installation
+
+```shell
+$ npm install event-store-adapter-js
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)